### ISSUE TITLE
Include package metadata in GH repo

### ISSUE THREAD

**Issue by nfultz** [YingfanWang/PaCMAP#25]
I'd like to use the current version from github, but there is no package metadata / setup.py checked in to the repo :(

Although it is definitely in the version on pypi.

Ideally, this would work:

```
pip install  git+https://github.com/YingfanWang/PaCMAP
```

**Comment by hyhuang00** [comment]
I see -- you're right, there is no package metadata in the repository for now. Currently I have some files called `pyproject.toml` and `setup.cfg` for packaging the code. Are these files what you want me to share? I'm happy to upload them to GitHub right now, but I'm not sure whether uploading them could achieve the behavior you expect.

There are some reasons for the current difference between the PyPI version and the one on GitHub: 1. The current version has the ability to transform other data, but the API is not compatible with the scikit-learn style (`transform()` method), so I'm trying to fix it soon. 2. I haven't tested the behavior under the new methods so I'm not sure whether I need to add some extra pairs for the new dataset.

**Comment by nfultz** [comment]
Yes, that would be nice if we got installing the dev version working. If you upload your pyproject and setup configs, I can send you a PR that gets installation directly from gh working later this week, not too complicated. 

I'm working with PacMAP on a client engagement and need the version with the Xp arguments. I already wrote a wrapper with a transform method to use for my project (I could share it if you want), but it has it's own problems:  pretty slow, requires original data set, and predictions are not rotationally invariant. All things I can deal with for now, though
